src/spoke/Spoke.sol

```diff
@@ -108,6 +108,7 @@ contract Spoke is Auth, Recoverable, ReentrancyProtection, ISpoke, ISpokeGateway
         uint256 cost =
             sender.sendInitiateTransferShares(centrifugeId, poolId, scId, receiver, amount, remoteExtraGasLimit);
         require(msg.value >= cost, NotEnoughGas());
+        if (msg.value > cost) gateway.withdrawSubsidy(poolId, msg.sender, msg.value - cost);
```

Comment:
If a FM forgets to set the Spoke as gateway manager, this will always revert. Not sure we should do this 🤔